livekit_models.proto

```diff
@@ -167,3 +167,48 @@ message ClientInfo {
   string browser = 7;
   string browser_version = 8;
 }
+
+// server provided client configuration
+message RTCClientConfiguration {
+	AudioConfiguration audio = 1;
+  VideoConfiguration video = 2;
+  SignalConfiguration signal = 4;
+}
+
+message AudioConfiguration {
+  message AudioProcessing {
+    // echo cancellation
+    bool aec = 1;
+    // noise suppression
+    bool ns = 2;
+    // automatic gain control
+    bool agc = 3;
+  }
+  message AudioPreferences {
+    bool dtx = 1;
+    int32 sample_rate = 2;
```

Comment:
I rather naively replaced nb/fwb/wb with a more generic sample_rate, not sure if that covers the exact same scenario?

Reply:
Yeah, `sample_rate` will cover everything. The trick there would be to check how browsers behave if given random sample rates. But, I am guessing we will never give random sample rate. So, keeping it as sample rate should be more flexible.

Reply:
generally (talking about browsers only) the way we request the preferences we set for certain tracks will *try* to get fullfilled by the browser. In cases we have a random sample rate that doesn't make sense, the brower *should* return the closest appropriate sample rate. I have tested this in the past for other constraints, but will have to confirm for `sampleRate`